docs/src/increase-observability/logs.md

```diff
@@ -6,6 +6,8 @@ aliases:
   - /development/logs.html
 ---
 
+{{% legacy-regions featureIntro="Deploy hook and cron activity logs" featureShort="these logs" plural=true %}}
```

Comment:
On this page, I thought it would get lost at the bottom. My idea was it would seem like I could ready only the activity logs section or only the container logs section and skip the rest. I don't think it's worth repeating for each section, so once at the top seemed to make sense to me.

The other places have just one place, so it was easier to fit into the structure there.